jax/_src/numpy/reductions.py

```diff
@@ -308,7 +376,13 @@ def any(a: ArrayLike, axis: Axis = None, out: None = None,
                      keepdims=keepdims, where=where)
 
 amin = min
-amax = max
+
+def amax(a: ArrayLike, axis: Axis = None, out: None = None,
```

Comment:
You'll need to list this as an alias in the test file, I think.

Reply:
Done.